src/applications/poketch/analog_watch/main.c

```diff
@@ -0,0 +1,232 @@
+#include <nitro.h>
+#include <nitro/sinit.h>
+
+#include "applications/poketch/analog_watch/graphics.h"
+#include "applications/poketch/poketch_button.h"
+#include "applications/poketch/poketch_system.h"
+
+#include "bg_window.h"
+#include "heap.h"
+#include "rtc.h"
+#include "sys_task.h"
+#include "sys_task_manager.h"
+#include "touch_screen.h"
+
+typedef struct PoketchAnalogWatch {
+    u8 state;
+    u8 subState;
+    u8 shouldExit;
+    u8 unused[2];
+    u8 lastUpdateMinute;
+    u8 brightnessChangePending;
+    u32 unused2[2];
+    PoketchButtonManager *buttonManager;
+    AnalogWatchData watchData;
+    AnalogWatchGraphics *graphics;
+    PoketchSystem *poketchSys;
+} PoketchAnalogWatch;
+
+enum AnalogWatchState {
+    STATE_LOAD_APP = 0,
+    STATE_UPDATE,
+    STATE_SHUTDOWN,
+};
+
+static void NitroStaticInit(void);
+
+static BOOL New(void **appData, PoketchSystem *poketchSys, BgConfig *bgConfig, u32 appID);
+static BOOL Init(PoketchAnalogWatch *appData, PoketchSystem *poketchSys, BgConfig *bgConfig, u32 appID);
+static void Free(PoketchAnalogWatch *appData);
+static void Exit(void *appData);
+static void TouchCallback(u32 buttonID, u32 buttonState, u32 touchState, void *appData);
+
+static void Task_Main(SysTask *task, void *taskMan);
+static void ChangeState(PoketchAnalogWatch *appData, enum AnalogWatchState newState);
+static BOOL State_LoadApp(PoketchAnalogWatch *appData);
+static BOOL State_UpdateApp(PoketchAnalogWatch *appData);
+static BOOL State_UnloadApp(PoketchAnalogWatch *appData);
+
+static const TouchScreenRect screenHitBox[] = {
+    { .rect = {
+          .top = POKETCH_SCREEN_MIN_Y,
+          .bottom = POKETCH_SCREEN_MAX_Y,
+          .left = POKETCH_SCREEN_MIN_X,
+          .right = POKETCH_SCREEN_MAX_X } }
+};
```

Comment:
*polish*: `sScreenHitBox`

---

**polish**: Add trailing commas to force `clang-format` to render this correctly:

```suggestion
static const TouchScreenRect screenHitBox[] = {
    { 
        .rect = {
            .top = POKETCH_SCREEN_MIN_Y,
            .bottom = POKETCH_SCREEN_MAX_Y,
            .left = POKETCH_SCREEN_MIN_X,
            .right = POKETCH_SCREEN_MAX_X,
        },
     },
};
```

Ordinarily, I'd prefer this to be in a tabular format, but this is fine since it's the only hitbox.